src/app/modules/submission/encrypt-submission/encrypt-submission.service.ts

```diff
@@ -184,6 +186,19 @@ export const performEncryptPostSubmissionActions = ({
         const webhookUrl = form.webhook?.url
         if (!webhookUrl) return okAsync(form)
 
+        // replace encryptedContent with encryptedWebhookContent before firing
+        if (encryptedWebhookContent)
+          submission.encryptedContent = encryptedWebhookContent
+        else {
+          logger.error({
+            message:
+              'Error while sending webhook, no encryptedWebhookContent found',
+            meta: {
+              action: 'useEncryptedWebhookContent',
```

Comment:
nit (non blocking): also, understand there are some inconsistencies in this function itself, but might want to consider using the `...logMeta`